### ISSUE TITLE
Blocking error

### ISSUE THREAD

**Issue by Zazanels** [Coyote21/FoodRemover#7]
https://i.imgur.com/pRcYyti.png 

What is blocking reason ?  Can't figure out. Thanks.

**Comment by Coyote21** [comment]
Wow I have not looked at this code in a long time... Fortunately I'm just now starting to get back into Skyrim.... But given how long it's been, how much I have forgotten about Synthesis patches, and that the screenshot you attached seems to be in French, I'm guessing? I'm not sure when I'll be able to fix this.

But I'm not seeing anything else that performs a similar function to what this patch does, and this is still something I want in my own Skyrim, so I'm sure I'll get to it again soon.

**Comment by Zazanels** [comment]
Hi yes French ! 

''IplaceObjectGetter does not contain a definition for Ownership and no accessible Ownership extension method was found that accepts a first argument of type IplaceObjectGetter (is a using directive or assembly reference missing?)''

**Comment by Coyote21** [comment]
OK You've sucked me back in.. I'm just beginning to get back up to speed but as a start, about a year ago, I made a newer version called Food-Remover-Plus that has updated code and more capabilities. I have fired Food-Remover-Plus up myself and got no errors. I have yet to examine the resulting patch in xEdit or test it in-game but using the newer Food-Remover-Plus is probably a good start.

Of course if that also has issues, post them in the Food-Remover-Plus issues. I'll close this one in a day or two if I don't hear from you sooner....
